README.md

```diff
@@ -113,12 +113,12 @@ srf.plot()
 A similar example but for a three dimensional field is exported to a [VTK](https://vtk.org/) file, which can be visualized with [ParaView](https://www.paraview.org/) or [PyVista](https://docs.pyvista.org) in Python:
 
 ```python
-from gstools import SRF, Gaussian
 import matplotlib.pyplot as pt
```

Comment:
Could we stick to ``plt`` as used in the whole matplotlib documentation?

Reply:
Why is matplotlib even imported here? We could kick it out of the README almost everywhere.

Reply:
done